arduino_protocols/r_active_avoidance/r_active_avoidance.ino

```diff
@@ -1,73 +1,110 @@
+// EXPERIMENTAL VARIABLES
 const int N_TRIALS = 20;
 unsigned long ACCLIMATION_DURATION = 20;                       // SECONDS
 unsigned long TONE_DURATION = 15;                              // SECONDS
 unsigned long SHOCK_DURATION = 1;                              // SECONDS
-int CS_FREQUENCY = 5000;                                  // IN HERTZ
+int CS_FREQUENCY = 5000;                                       // IN HERTZ
 int ITI_INTERVALS[] = {40, 60, 80, 100, 120};                  // list of the inter-trial-intervals: ITI
-
-// LOCATION VARIABLES.
-// ########################################################
+unsigned long MOTION_DETECTION_DURATION = 30;                  // SECONDS
+//##################################################################################################################
+//##################################################################################################################
+//##################################################################################################################
+//##################################################################################################################
+//##################################################################################################################
+// LOCATION VARIABLES
 int LEFT_ACTIVE;                                        // HIGH IF A COMPARTMENT IS ACTIVE, ELSE LOW
 int RIGHT_ACTIVE;
-
-// ######################################
+//##################################################################################################################
+// TIMING VARIABLES
+// For control of US and CS
 unsigned long CURRENT_TONE_DELAY;
 unsigned long START_TONE;
 unsigned long DELTA_TONE_SHOCK = TONE_DURATION - SHOCK_DURATION;
 unsigned long ITI_DURATION;
 
+// For control of motion detection
+unsigned long MOTION_DETECTION_START;
+unsigned long MOTION_DETECTION_CURR;
+//##################################################################################################################
+// VARIABLES FOR STATISTICS
+unsigned long ESCAPE_LATENCY_START;
+unsigned long ESCAPE_LATENCY_END;
+unsigned long ESCAPE_LATENCY_DELTA;
+float ESCAPE_LATENCY_CUMULATIVE;
+//##################################################################################################################
+// SESSION VARIABLES
+int TOTAL_AVOIDANCE_SUCCESS = 0;                           // CUMULATIVE COUNT OF SUCCESSFUL AVOIDANCE RESPONSES
+int TOTAL_AVOIDANCE_FAILURE = 0;                           // CUMULATIVE COUNT OF FAILED AVOIDANCE RESPONSES
+//##################################################################################################################
+
+
+/*
+    PIN ASSIGNMENTS
+*/
+//##################################################################################################################
 // DIGITAL PINS
-// ########################################################
+// Speakers
 const int speaker_pin = 3;
```

Comment:
This line becomes obsolete because from now on, only buzzers will be used to generate sound.
Action required: @audreyyin9877 
- [ ] remove line 64-65